partner_industry_secondary/models/res_partner_industry.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2015 Antiun Ingenieria S.L. - Javier Iniesta
```

Comment:
@cristinamartinrod I think here to add the only company name and  Contributors move  `CONTRIBUTORS` file

```
# Copyright 2015 Antiun Ingenieria S.L.
# Copyright 2019 Tecnativa S.L.
# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
```

Improve in all file

Reply:
Copyright and contributors are not exactly the same.